player.html

```diff
@@ -1,130 +1,154 @@
                 </div>
-            </a>
+              </div>
 
-            <a href="">
-                <div class="suggest7">
-                    <img src="images/7. Databases.jpg" height="200" width="300" alt="Servers">
-                    <p class="suggest-head7"><b>Database Essentials</b></p>
-                    <p class="desc7">Learn different database technologies and decide the best one for your <b>Read More</b></p>
-                </div>
-            </a>
+               <div class="description">
+                 <p><b>16 views  Dec 31, 2022</b> <br> In this tutorial, we show you more on Tag Attributes, how to add a favicon and how to create lists in html.</p>
+               </div> 
+               <div class="commenter">
+                <input class="comment-text" type="text" placeholder="Add a comment...">
+                <img class="comment-image" src="images/My Pic.jpg" height="200" width="300" alt="Picture">
+               </div>
 
-            <a href="">
-                <div class="suggest8">
-                    <img src="images/8. Empower Women.jpg" height="200" width="300" alt="Empower Women">
-                    <p class="suggest-head8"><b>Women in Technology</b></p>
-                    <p class="desc8">Empower women to join the world of technology, make an impact and prove <b>Read More</b></p>
-                </div>
-            </a>
+                <hr class="separator">
+                <button class="comment-button" type="button"><a href="">Comment</a></button>
+                <button class="comment-button" type="button"><a href="">Cancel</a></button>
+            </section>
 
-            <a href="">
-                <div class="suggest9">
-                    <img src="images/9. Networking.jpg" height="200" width="300" alt="Networking">
-                    <p class="suggest-head9"><b>Networking Essentials</b></p>
-                    <p class="desc9">Learn networking and merge that into your professional career development <b>Read More</b></p>
-                </div>
-            </a>
+            <section class="sidebar">
+                <h6>Suggested Videos...</h6>
 
-            <a href="">
-                <div class="suggest10">
-                    <img src="images/10. Cable Handling.jpg" height="200" width="300" alt="Network Cable">
-                    <p class="suggest-head10"><b>Cable Handling</b></p>
-                    <p class="desc10">Learn the proper way to handle messy network cables to make your work easier <b>Read More</b></p>
-                </div>
-            </a>
-        </div>
-
-        <!--
-        <iframe class="video" width="560" height="315" src="https://www.youtube.com/embed/AO_PNnBETJI" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
-            -->
```

Comment:
:clap: Great effort :muscle: attempting to experiment with different ways to utilize the iframe feature. However, you could consider removing commented code from your codebase. Leaving commented codes in your codebase has the tendency of making your code clumsy, untidy and difficult to read.